model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+  - id: metric.otel.sdk.span.ended_count
+    type: metric
+    metric_name: otel.sdk.span.ended_count
+    stability: development
+    brief: "The number of created spans for which the end operation was called"
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+
+  - id: metric.otel.sdk.span.processor.queue_size
+    type: metric
+    metric_name: otel.sdk.span.processor.queue_size
+    stability: development
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: otel.sdk.span.processor.queue_capacity
+    stability: development
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.processor.spans_processed
+    type: metric
+    metric_name: otel.sdk.span.processor.spans_processed
+    stability: development
+    brief: "The number of spans for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` must be empty. For failed processing, `error.type` must contain the failure cause.
+      For the SDK Simple and Batching Span Processor a span is considered to be processed already when it has been submitted to the exporter, not when the corresponding export call has finished.
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching Span Processors MUST use `queue_full` for spans dropped due to a full queue.
+        examples: ["queue_full"]
+
+  - id: metric.otel.sdk.span.exporter.spans_inflight
+    type: metric
+    metric_name: otel.sdk.span.exporter.spans_inflight
+    stability: development
+    brief: "The number of spans which were passed to the exporter, but that have not been exported yet (neither successful, nor failed)"
+    note: |
+      For successful exports, `error.type` must be empty. For failed exports, `error.type` must contain the failure cause.
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.exporter.spans_exported
```

Comment:
```suggestion
  - id: metric.otel.sdk.exporter.span.exported.count
```

Reply:
Done in [48b62c8](https://github.com/open-telemetry/semantic-conventions/pull/1631/commits/48b62c832f5fab602cfd07106a5ccfcafa1ecdf9)